src/features/surveys/l10n/messageIds.ts

```diff
@@ -152,7 +152,7 @@ export default makeMessages('feat.surveys', {
     lastNameColumn: m('Last name'),
     link: m('Link'),
     personRecordColumn: m('Respondent'),
-    suggestedPeople: m('Suggested people'),
+    suggestedPeople: m('Other linked respondents'),
```

Comment:
Why this change? We're looking for people who match the specific person here, not other respondents as in the case of views?

Reply:
You are right, I changed it again! I misunderstood what the suggested people was! thanks for pointing out!